scout/server/blueprints/cases/forms.py

```diff
@@ -3,16 +3,44 @@
 
 from flask_wtf import FlaskForm
 from wtforms import (
-    BooleanField,
-    DecimalField,
     Field,
     TextField,
+    SelectField,
     SelectMultipleField,
     IntegerField,
     SubmitField,
+    BooleanField,
+    validators,
 )
 from wtforms.widgets import TextInput
 
+CASE_SEARCH_KEY = [
+    "Case or individual name",
+    "HPO term",
+    "Search synopsis",
+    "Gene panel",
+    "Case status",
+    "Phenotype group",
+    "Patient cohort",
+    "Similar case",
+    "Similar phenotype",
+    "Pinned gene",
+    "Causative gene",
+    "Assigned user",
+]
+
+
+class CaseFilterForm(FlaskForm):
```

Comment:
This was entirely written in the HTML before. I think it's better to have this as a Flask-wft object because it can be reused, for instance in the slice query of the dashboard.